client/components/plans/plan/index.jsx

```diff
@@ -83,6 +88,11 @@ module.exports = React.createClass( {
 				'is-plan-specific': feature.planSpecific
 			} );
 
+			if ( abtest( 'plansFeatureList' ) === 'andMore' && feature.testVariable ) {
```

Comment:
Updated to use `isBusiness` here: https://github.com/Automattic/wp-calypso/pull/3316/files#diff-c32b54022cfc1b81d8fc3eb261ceb50cR104